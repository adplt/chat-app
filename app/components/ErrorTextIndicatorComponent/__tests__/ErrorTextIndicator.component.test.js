import React from 'react';
import renderer from 'react-test-renderer';
import ErrorTextIndicator from '../ErrorTextIndicator.component';

describe('ErrorTextIndicator component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <ErrorTextIndicator />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
