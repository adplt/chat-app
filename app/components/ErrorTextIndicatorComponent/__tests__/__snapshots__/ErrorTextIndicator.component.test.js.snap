// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorTextIndicator component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginBottom": 5,
      "marginTop": 5,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ED1D25",
        "fontSize": 14,
        "fontWeight": "600",
      }
    }
  />
</View>
`;
